TKGame/Content/Weapons/Weapon.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TKGame.Content.Weapons
```

Comment:
This is exactly what I plan to do with the Item class. Make it an abstract class to use the strategy pattern.